addons/account/views/account_account_views.xml

```diff
@@ -142,7 +142,7 @@
             <field name="model">account.account</field>
             <field name="arch" type="xml">
                 <search string="Accounts">
-                    <field name="name" filter_domain="['|', ('name', 'ilike', self), ('code', '=ilike', self + '%')]" string="Account"/>
+                    <field name="name" filter_domain="['|', '|', ('name', 'ilike', self), ('code', '=ilike', self + '%'), ('placeholder_code', '=ilike', self + '%')]" string="Account"/>
```

Comment:
Do we always wish to search on the placeholder here ? Not only when the code is empty ?

Reply:
Yes :smile: This is because the 'placeholder code' is another useful information about the account. 
Note that code and placeholder code will be the same if the account belongs to the active company.
However, if the account does not belong to the active company, the placeholder code will reflect the account's actual code in a company it does belong to. That's quite useful for searching! :smile: